components/Banner.tsx

```diff
@@ -0,0 +1,35 @@
+import { Center, Flex, Text, chakra } from '@chakra-ui/react'
+
+const Banner = () => {
+  return (
+    <Center
+      py="2"
+      px="3"
+      bgGradient="linear(to-r,cyan.700, purple.500)"
+      color="white"
+      textAlign="center"
+    >
+      <Flex align="center" fontSize="sm">
+        <Text fontWeight="medium" maxW={{ base: '32ch', md: 'unset' }}>
+          Join our newsletter for exclusive updates!
```

Comment:
Update message to something like...

Acamdey V2 is coming soon 👀... Subscribe to our newsletter for updates!

Reply:
@kempsterrrr @wolovim 

Implemented y'alls suggestion.